src/photos/ducks/albums/index.jsx

```diff
@@ -25,13 +24,31 @@ const ALBUMS_QUERY = client =>
 export const ALBUM_QUERY = (client, ownProps) =>
   client.get(DOCTYPE_ALBUMS, ownProps.router.params.albumId).include(['photos'])
 
+const addPhotos = async (album, photos) => {
+  try {
+    const photoCountBefore = album.photos.data.length
+    await album.photos.addById(photos.map(({ _id }) => _id))
+    const photoCountAfter = album.photos.data.length
+    if (photoCountBefore + photos.length !== photoCountAfter) {
+      Alerter.info('Alerter.photos.already_added_photo')
+    } else {
+      Alerter.success('Albums.add_photos.success', {
+        name: album.name,
+        smart_count: photos.length
+      })
+    }
+  } catch (error) {
+    Alerter.error('Albums.add_photos.error.reference')
+  }
```

Comment:
I know this is not part of this PR, but I think that our `duck` functions, should do nothing related to the UI. Even if it's only an alert. WDYT?

Reply:
I'm not too sure. Yes, they shouldn't do major UI work, but alerts..?

This code is more or less like a redux action. So yeah, you wouldn't have much UI code there, but you could have things like `dispatch(alertError('message'))`. And that's pretty much what `Alerter.error()` does. But if you have an idea of a more suitable place to trigger the alerts, I'm happy to consider it !